sui-execution/latest/sui-adapter/src/static_programmable_transactions/execution/context.rs

```diff
@@ -1313,3 +1371,611 @@ unsafe fn create_written_object<Mode: ExecutionMode>(
         )
     }
 }
+
```

Comment:
From here down in this file it's all just copied over from the old programmable transactions codebase (since there were some external references from the static ptbs out into it).